content/slice/language-guide/class-types.md

```diff
@@ -210,35 +211,50 @@ For example:
 ```slice
 class CarPart {
     id: string
+    revision: int32
     tag(1) shippingWeight: float64?
 }
 ```
 
 ```csharp
 public partial class CarPart : SliceClass
 {
-    public string Id;
-    public double? ShippingWeight;
+    public required string Id { get; set; }
+
+    public int Revision { get; set; }
+
+    public double? ShippingWeight { get; set; }
+
+    // Parameterless constructor
+    public CarPart()
+    {
+    }
 
     // Primary constructor
-    public CarPart(string id, double? shippingWeight)
+    public CarPart(
+        string id,
+        int revision,
+        double? shippingWeight)
     {
-       ...
+        this.Id = id;
+        this.Revision = revision;
+        this.ShippingWeight = shippingWeight;
     }
 
     // Secondary constructor
-    public CarPart(string id)
+    public CarPart(string id, int revision)
     {
-       ...
+        this.Id = id;
+        this.Revision = revision;
     }
-
 }
 ```
 
 {% /aside %}
 
-The mapped class has a primary constructor which sets all the fields. If any field has an optional type, the mapped
-class has a second constructor with a parameter for each non-nullable C# field.
+The mapped class always provides a parameterless constructor and a primary constructor which sets all the fields. If
+any field has an optional type, the mapped class also provides a secondary constructor with a parameter for each
+non-nullable C# field.
 
 Slice class inheritance maps the C# class inheritance as you would expect:
```

Comment:
Not in your PR, but this sentence sounds wrong. Maybe it's supposed to be?:
"Slice class inheritance maps *_to_* C# class inheritance as you would expect"
____________________________.^^

Reply:
Fixed